atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraReloadableKvsConfig.java

```diff
@@ -42,21 +42,19 @@ public CassandraReloadableKvsConfig(
 
     @Override
     public CassandraServersConfigs.CassandraServersConfig servers() {
-        CassandraServersConfig servers = chooseServers();
+        CassandraServersConfig servers;
+        // get servers from install config (for backcompat), otherwise get servers from runtime config
+        if (config.servers().numberOfThriftHosts() > 0) {
+            servers = config.servers();
+        } else {
+            servers = chooseConfig(CassandraKeyValueServiceRuntimeConfig::servers, config.servers());
+        }
```

Comment:
```suggestion
        // get servers from install config (for backcompat), otherwise get servers from runtime config
        if (config.servers().numberOfThriftHosts() > 0) {
            return config.servers();
        }

        CassandraServersConfig servers = chooseConfig(CassandraKeyValueServiceRuntimeConfig::servers, config.servers());
```